srcpkgs/samba/patches/getpwent_r.patch

```diff
@@ -0,0 +1,80 @@
+diff --git source4/torture/local/nss_tests.c source4/torture/local/nss_tests.c
+index 2cd6122..04f13c6 100644
+--- source4/torture/local/nss_tests.c
++++ source4/torture/local/nss_tests.c
+@@ -247,7 +247,6 @@ static bool test_getgrnam_r(struct torture_context *tctx,
+ 	return true;
+ }
+ 
+-
+ static bool test_getgrgid(struct torture_context *tctx,
+ 			  gid_t gid,
+ 			  struct group *grp_p)
```

Comment:
Doesn't do anything, as far as I can see? Removing now should help future rebasing...

Reply:
This one is straight from alpine as well, so IMO it's fine to leave as is.